### DISCUSSION TITLE
Custom 1-tile vehicle with one wheel indicates "Wheels: lack"

### DISCUSSION THREAD

**Discussion by tsulh** [CleverRaven/Cataclysm-DDA#48082] [Q&A] [answered]
The vehicle has one wheel, but the indicator says "Wheels: lack". Please see the screenshots. Is it a bug or a wrong vehicle design? The wheel on either model also doesn't make the dragging easier, the speed of dragging the wheelbarrow is the same either with or without a wheel. The speed is slower when the wheelbarrow is loaded with logs.
https://i.imgur.com/H596jkq.png
https://i.imgur.com/iPq2bPE.png

**Comment by anothersimulacrum** [comment, marked as answer]
Both those wheels require another wheel to work. Wheels such as casters or wheelchair wheels will work without another wheel tile.

**Reply by tsulh** [reply]
> Both those wheels require another wheel to work.

Should this be added in the end of the description of each such wheel? E.g., "Requires at least one other wheel to function." This is not obvious without a note, especially given irl unicycles.

**Reply by anothersimulacrum** [reply]
That'd be good. I'll look into that.
